attributes/defaults.rb

```diff
@@ -54,6 +60,14 @@
   default['mysqld']['my.cnf']['mysqld_safe']['skip_log_error'] = true
 end
 
+# Password columns for user passwords
+default['mysqld']['pwd_col'] = case node['mysqld']['db_install']
```

Comment:
Would it make sense to document the `pwd_col` attribute in the README? It could be overridden using `override['mysqld']['pwd_col']` if necessary.

It might be a good idea to keep this internal, though.